charactersheet/charactersheet/templates/dm/encounter_sections/monster_section.tmpl.html

```diff
@@ -0,0 +1,666 @@
+                       class="form-control"
+                       placeholder="22"
+                       data-bind='value: blankMonster().challenge'>
+              </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Experience</label>
+              <div class="col-sm-10">
+                <input type="number"
+                       class="form-control"
+                       placeholder="11500"
+                       data-bind='value: blankMonster().experience'>
+              </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Description</label>
+              <div class="col-sm-10">
+                <textarea type="text"
+                       class="form-control" rows="4"
+                       placeholder="Multiattack. The dragon can use its Frightful Presence. It then makes three attacks..."
+                       data-bind='value: blankMonster().description'></textarea>
+              </div>
+          </div>
+          <div class="modal-footer">
+            <button type="button"
+                    class="btn btn-primary"
+                    data-bind='click: addMonster'
+                    data-dismiss="modal">Add</button>
+          </div>
+        </form>
+      </div> <!-- Modal Body -->
+    </div> <!-- Modal Content -->
+  </div> <!-- Modal Dialog -->
+</div> <!-- Modal Fade -->
+
+<!-- ViewEdit Modal -->
+<div class="modal fade"
+     id="viewWeapon"
+     tabindex="-1"
+     role="dialog"
+     data-bind="modal: {
+      open: selecteditem,
+      onopen: modalFinishedOpening,
+      onclose: modalFinishedClosing
+    }">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button"
+                class="close"
+                data-dismiss="modal"
+                aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <h4 class="modal-title">Edit a Monster</h4>
+      </div>
+      <div class="modal-body">
+        <!-- Begin Tabs -->
+        <ul class="nav nav-tabs tabs">
+          <li role="presentation" data-bind="click: selectPreviewTab, css: previewTabStatus">
+            <a href="#" aria-controls="itemsModalPreview" role="tab" data-toggle="tab">
+                <b>Preview</b>
+            </a>
+          </li>
+          <li role="presentation" data-bind="click: selectEditTab, css: editTabStatus">
+            <a href="#" aria-controls="itemsModalEdit" role="tab" data-toggle="tab">
+                <b>Edit</b>
+            </a>
+          </li>
+        </ul>
+        <div class="tab-content" data-bind="with: selecteditem">
+          <div role="tabpanel" data-bind="css: $parent.previewTabStatus" class="tab-pane">
+            <div class="h3">
+              <span data-bind="text: name"></span>&nbsp;
+              <small data-bind="text: type"></small>
```

Comment:
Can you make a label method that does this text construction? This is too much view state imo.

```js
self.nameTypeLabel = ko.pureComputed(function() {
  var text = self.name();
  if (someCriteria) {
    text += ', ' + type;
  } 
  return text;
});
```